### PR TITLE
setup-CI

### PR DESCRIPTION
**Pull request by bean00**
I believe you'll have to set the value for the `$HEROKU_API_KEY` on your end, Jason (in the settings for Travis CI)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
+[![Build Status](https://travis-ci.org/jdesrosiers/todo-api.svg?branch=master)](https://travis-ci.org/jdesrosiers/todo-api)
+
 TODO API
 ========
 
 The TODO API was created to showcase a Hypermedia API.
 
+**Note**: This application is deployed to Heroku, at https://hypermedia-todo.herokuapp.com/
```
Comment (jdesrosiers) [inline README.md]: I was hoping for a little more that this. Deployment deserves it's own section in the README. If I want to know how the app is deployed, this should answer all my questions. Mention that we are using a continuous deployment strategy and what that means. Mention that all changes are immediately and automatically deployed as soon as they are merged to master. Mention that deployment should never be done manually, only through the automated process.

**Review by jdesrosiers — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bean00) [reply README.md]: @jdesrosiers, changes have been made. Please let me know what other comments you have.

File: README.md [hunk 2]
```diff
@@ -16,3 +18,18 @@ What, no tests?! This is sacrilege!
 
 Resourceful allows us to declaratively define an API with almost no code. Until
 there is need for actual code, we will put off setting up a test framework.
+
+Deployment
+----------
+
+Continuous deployment is used for this application. Continuous deployment aims to automate the process of deploying code. Each stage in the process is automated, so that every change that passes all the tests is deployed to production automatically. These are the steps in the process:
```
Comment (bean00) [inline README.md]: @jdesrosiers, changes have been made. Please let me know what other comments you have.

File: README.md (hunk 1 above)
Comment (bean00) [reply README.md]: @jdesrosiers, changes have been made. Please let me know what other comments you have.

**Review by jdesrosiers — APPROVED** [review]

**Pull request merged**